Unable to compile CRAS

I have been trying to create a chroot with target `audio` and I get this error
```
Compiling CRAS (i386)...
In file included from libcras/cras_client.c:47:
libcras/cras_client.h:1439:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1454:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1475:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1491:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1506:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1530:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1550:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1570:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1592:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1608:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1623:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1638:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1653:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1668:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1683:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1732:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1753:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1767:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1784:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1799:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1815:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1831:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1847:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1878:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1893:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1908:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1924:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1940:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1955:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1970:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1985:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:2000:1: error: 'cfi-icall' attribute directive ignored [-Werror=attributes]
 {
 ^
cc1: all warnings being treated as errors
Failed to complete chroot setup.
```

I tried changing the releases to `xenial, bionic, focal, sid, stretch` and none of them worked. I even tried `gcc-7, gcc-9, gcc-10, gcc-11` and still the same error. If `-Wno-error=attributes` flag is added to the `make` target, this issue can be resolved.
Any help would be greatly appreciated!

--- a/targets/audio
+++ b/targets/audio
@@ -116,8 +116,7 @@ build_cras() {
         tar -xmf adhd.tar.gz cras/src
 
         cd cras/src
-	# Fixes #4506
-	sed -i 's/-Werror//g' Makefile.am
+
         # Create version file
         echo '#define VCSID "crouton-'"$ADHD_HEAD"'"' > common/cras_version.h
 
@@ -176,8 +175,8 @@ END
             # Note: -Wno-int-in-bool-context is necessary for building CRAS on
             # older ChromeOS versions (issue #3352).
             echo '
-                CFLAGS="$CFLAGS -Wno-int-in-bool-context -DCRAS_SOCKET_FILE_DIR=\"/var/run/cras\""
-
+                CFLAGS="$CFLAGS -Wno-int-in-bool-context -DCRAS_SOCKET_FILE_DIR=\"/var/run/cras\" -Wno-attributes"
+		
                 buildlib libcras
 
                 # Pass -rpath=$CRASLIBDIR to linker, so we do not need to add

--- a/targets/audio
+++ b/targets/audio
@@ -116,7 +116,8 @@ build_cras() {
         tar -xmf adhd.tar.gz cras/src
 
         cd cras/src
-
+	# Fixes #4506
+	sed -i 's/-Werror//g' Makefile.am
         # Create version file
         echo '#define VCSID "crouton-'"$ADHD_HEAD"'"' > common/cras_version.h
 

